iroh-ctl/src/store.rs

```diff
@@ -58,10 +56,15 @@ pub async fn run_command(rpc: Client, cmd: Store) -> Result<()> {
         StoreCommands::Block(block) => match block.command {
             BlockCommands::Get { cid } => {
                 let b = rpc.try_store()?.get(cid).await?;
-                println!("{:?}\n", b);
+                if let Some(b) = b {
+                    std::io::stdout().write_all(&b)?;
+                } else {
+                    println!("local store does not contain block {}", cid);
```

Comment:
convert all `println` into `debug`, `error` or other where appropriate and not just the plain CLI output? Sometimes its good to just use the CLI and pipe stuff around ignoring verbose logs and basically relying on the exit code. Also would be nice for these error cases to just `exit(1)`

Reply:
I fully agree we need better erroring, but for now I'm noting down the issue and keeping most of the printlns for the areas where we really are just trying to print to stdout. Removing this particular println, tho.